src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/GenerateRepresentedClaimCopyTest.java

```diff
@@ -2,58 +2,63 @@
 
 import org.junit.Test;
 import org.springframework.http.HttpHeaders;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.web.servlet.ResultActions;
-import uk.gov.hmcts.cmc.claimstore.BaseIntegrationTest;
+import uk.gov.hmcts.cmc.claimstore.DocumentManagementBaseIntegrationTest;
+import uk.gov.hmcts.cmc.claimstore.exceptions.DocumentManagementException;
 import uk.gov.hmcts.cmc.domain.models.Claim;
 import uk.gov.hmcts.cmc.domain.models.sampledata.SampleAmountRange;
 import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
 import uk.gov.hmcts.reform.cmc.pdf.service.client.exception.PDFServiceClientException;
 
+import java.util.List;
+
 import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-public class GenerateRepresentedClaimCopyTest extends BaseIntegrationTest {
+@TestPropertySource(
+    properties = {
+        "feature_toggles.document_management=true"
+    }
+)
+public class GenerateRepresentedClaimCopyTest extends DocumentManagementBaseIntegrationTest {
 
-    private static final byte[] PDF_BYTES = new byte[]{1, 2, 3, 4};
     private static final String AUTH_TOKEN = "Bearer authDataString";
 
     @Test
-    public void shouldReturnPdfDocumentIfEverythingIsFine() throws Exception {
+    public void shouldReturnPdfDocumentFromDocumentStoreIfEverythingIsFine() throws Exception {
```

Comment:
Great you test the case where PDF is missing and needs an upload but other scenario where document exists and just needs a download.

Reply:
actioned